### PR TITLE
Release/0.3.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

**Review by bryanvullo — APPROVED** [review]
Looks good to me!

File: template.tpl [hunk 1]
```diff
@@ -619,86 +683,171 @@ const spDefaultCustomDefs = {
...
+/*
+ * Assumes logType argument is string.
+ * Determines if logging is enabled.
+ *
+ * @param logType {string} - the logType set ('no', 'debug', 'always')
+ * @returns - whether logging is enabled (boolean)
+ */
+const determineIsLoggingEnabled = (logType) => {
+  const containerVersion = getContainerVersion();
+  const isDebugMode = containerVersion ? containerVersion.debugMode : false;
```
Comment (Bukashk0zzz) [inline template.tpl]: I think adding previewMode check like you mentioned earlier is good idea. As inside the UI we write "debug and preview"

**Comment by Bukashk0zzz** [comment]
[adatzer](https://github.com/adatzer) I liked your setting of js prettifier. Can you please share them?

File: template.tpl (hunk 1 above)
Comment (adatzer) [reply template.tpl]: Done!

**Comment by adatzer** [comment]
Thank you very much for reviewing @Bukashk0zzz ! 

For prettifier just using [Prettier](https://prettier.io/docs/en/install.html) locally (after extracting code from the template). Only setting is for single quotes, e.g.

```
npx prettier --single-quote --write templateCode.js
```

**Review by paulboocock — APPROVED** [review]

**Comment by Bukashk0zzz** [comment]
Finally public Logs standard description is here @adatzer @paulboocock  https://github.com/stape-io/gtm-standards/blob/main/standards/GTMS-3.md

If you have any feedback on this feel free to comment. Thanks.

**Pull request merged**